packages/react/README.md

```diff
@@ -1,205 +1,152 @@
-*Use of this software is subject to important terms and conditions as set forth in the License file*
+# ReactJS Scaffold Proposal
```

Comment:
```suggestion
# ReactJS Scaffod
```

Reply:
Actually - I think we probably need to re-write the README. It mostly reads as a proposal. Could you revert to the old version and update anything that's changed?

Reply:
Sure i will work in this change